_rules/html-page-lang-b5c3f8.md

```diff
@@ -19,30 +19,25 @@ authors:
 
 ## Applicability
 
-The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element.
-
-**Note**: Documents embedded into other documents, such as through `iframe` or `object` elements are not applicable because they are not web pages according to the definition in WCAG.
+This rule applies to any [document element](https://dom.spec.whatwg.org/#document-element) if it is an `html` element that has a [node document](https://dom.spec.whatwg.org/#concept-node-document) with a [content type](https://dom.spec.whatwg.org/#concept-document-content-type) of `text/html` and whose [browsing context](https://html.spec.whatwg.org/#browsing-context) is a [top-level browsing context](https://html.spec.whatwg.org/#top-level-browsing-context).
```

Comment:
Can we simplify the language here? You've made this technically precise, but it has made this much harder to read. Perhaps:

> The rule applies to any `html` element that is in a [document] where the [content type] is `text/html`, and not embedded in an iframe or object (e.g. in the [top-level browser context]).

Reply:
> The rule applies to any html element that is in a [document] where the [content type] is text/html, and not embedded in an iframe or object (e.g. in the [top-level browser context]).

The above is saying something entirely different though, and would mean that the following code has two applicable `<html>` elements:

```html
<!doctype html>
<html>
  <body>
    <script>
      document.body.appendChild(document.createElement("html"));
    </script>
  </body>
</html>
```

Only one of those `<html>` elements is a _document element_; there's a reason for the technically precise language used. I should be able to avoid the "stuttering" use of "browsing context" though, if that helps.